Ginger/Ginger/Run/RunSetActions/ExportResultsToALMConfigPage.xaml.cs

```diff
@@ -160,7 +168,7 @@ private async void xExportToALMBtn_Click(object sender, RoutedEventArgs e)
                 }
             });
             IsProcessing = false;
-            xExportToALMLoadingIcon.Visibility = Visibility.Collapsed;
```

Comment:
Remove xExportToALMLoadingIcon as it is not being used anymore

Reply:
Removed